paasta_tools/api/views/instance.py

```diff
@@ -115,6 +116,45 @@ def chronos_instance_status(
     return cstatus
 
 
+def tron_instance_status(
+    instance_status: Mapping[str, Any],
+    service: str,
+    instance: str,
+    verbose: bool,
+) -> Mapping[str, Any]:
+    status: Dict[str, Any] = {}
+    client = tron_tools.get_tron_client_for_cluster(settings.cluster)
+    short_job, action = instance.split('.')
+    job = f"{service}.{short_job}"
+
+    job_content = client.get_job_content(job=job)
+    latest_run_id = client.get_latest_job_run_id(job_content=job_content)
+    action_run = client.get_action_run(job=job, action=action, run_id=latest_run_id)
+
+    # job info
+    status['job_name'] = short_job
+    status['job_status'] = job_content['status']
+    status['job_schedule'] = '{} {}'.format(job_content['scheduler']['type'], job_content['scheduler']['value'])
+    status['job_url'] = client.master_url + f'/api/jobs/{job}/{latest_run_id}'
```

Comment:
The master URL won't work for prod. You need the URLs that go through the jumpbox (e.g. tronweb-norcal-prod). I'd suggest you add a separate dashboard link for Tron, with the tronweb addresses.